.github/workflows/validation-nightly.yaml

```diff
@@ -0,0 +1,57 @@
+name: Snapshots Validation
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * *'
+jobs:
+  snapshot-builds:
+    strategy:
+      fail-fast: false
+      matrix:
+        hilla: ["2.2-SNAPSHOT", "2.1-SNAPSHOT", "1.3-SNAPSHOT"]
```

Comment:
Any possibility to fetch this automatically ?

Reply:
Not sure if it is easy. Maybe downloading and processing metadata. But I would start easy hard coding values and create a ticket for improvement